composer.json

```diff
@@ -25,44 +25,44 @@
         "symfony/asset": "self.version",
         "symfony/browser-kit": "self.version",
         "symfony/class-loader": "self.version",
-        "symfony/config": "2.99.99",
-        "symfony/console": "2.99.99",
+        "symfony/config": "self.version",
+        "symfony/console": "self.version",
         "symfony/css-selector": "self.version",
-        "symfony/dependency-injection": "2.99.99",
+        "symfony/dependency-injection": "self.version",
         "symfony/debug": "self.version",
         "symfony/debug-bundle": "self.version",
-        "symfony/doctrine-bridge": "2.99.99",
+        "symfony/doctrine-bridge": "self.version",
         "symfony/dom-crawler": "self.version",
-        "symfony/event-dispatcher": "2.99.99",
-        "symfony/expression-language": "2.99.99",
-        "symfony/filesystem": "2.99.99",
+        "symfony/event-dispatcher": "self.version",
+        "symfony/expression-language": "self.version",
+        "symfony/filesystem": "self.version",
         "symfony/finder": "self.version",
         "symfony/form": "self.version",
-        "symfony/framework-bundle": "2.99.99",
+        "symfony/framework-bundle": "self.version",
         "symfony/http-foundation": "self.version",
         "symfony/http-kernel": "self.version",
         "symfony/intl": "self.version",
         "symfony/monolog-bridge": "self.version",
         "symfony/options-resolver": "self.version",
-        "symfony/process": "2.99.99",
+        "symfony/process": "self.version",
         "symfony/property-access": "self.version",
         "symfony/proxy-manager-bridge": "self.version",
         "symfony/routing": "self.version",
-        "symfony/security": "2.99.99",
+        "symfony/security": "self.version",
```

Comment:
This change is blocked until https://github.com/doctrine/DoctrineCacheBundle/pull/48 is merged